sdk/communication/communication-sms/review/communication-sms.api.md

```diff
@@ -9,11 +9,53 @@ import type { KeyCredential } from '@azure/core-auth';
 import type { OperationOptions } from '@azure/core-client';
 import type { TokenCredential } from '@azure/core-auth';
 
+// @public
+export interface AddOptions {
+}
+
+// @public
+export interface CheckOptions {
+}
+
+// @public
+export interface OptOutAddResult {
+    errorMessage?: string;
+    httpStatusCode: number;
+    to: string;
+}
+
+// @public
+export interface OptOutCheckResult {
+    errorMessage?: string;
+    httpStatusCode: number;
+    isOptedOut: boolean;
+    to: string;
+}
+
+// @public
+export interface OptOutRemoveResult {
+    errorMessage?: string;
+    httpStatusCode: number;
+    to: string;
+}
+
+// @public
+export interface OptOutsClient {
+    add(from: string, to: string[], options?: AddOptions): Promise<OptOutAddResult[]>;
+    check(from: string, to: string[], options?: CheckOptions): Promise<OptOutCheckResult[]>;
+    remove(from: string, to: string[], options?: RemoveOptions): Promise<OptOutRemoveResult[]>;
+}
+
+// @public
+export interface RemoveOptions {
```

Comment:
Please have AddOptions/CheckOptions/RemoveOptions extending OperationOptions, similar to `SmsClient.SmsSendOptions`.

Reply:
Added extending OperationOptions to AddOptions/CheckOptions/RemoveOptions